launch/tier4_localization_launch/launch/util/util.launch.xml

```diff
@@ -11,11 +11,14 @@
   <arg name="use_intra_process" default="true" description="use ROS2 component container communication"/>
 
   <!-- pose_initializer -->
-  <include file="$(find-pkg-share pose_initializer)/launch/pose_initializer.launch.xml"/>
-
+  <group>
+    <include file="$(find-pkg-share pose_initializer)/launch/pose_initializer.launch.xml"/>
+  </group>
   <!-- util -->
-  <include file="$(find-pkg-share tier4_localization_launch)/launch/util/util.launch.py">
-    <arg name="input/pointcloud" value="$(var input/pointcloud)"/>
-    <arg name="output/pointcloud" value="$(var output/pointcloud)"/>
-  </include>
+  <group>
+    <include file="$(find-pkg-share tier4_localization_launch)/launch/util/util.launch.py">
```

Comment:
This include need two args of `container` and `use_intra_process` because of using `<group>`.
If not, they would not changed in util.launch.py.